content/blog/2018-05-10-onboarding-social-weather.md

```diff
@@ -0,0 +1,698 @@
+    ## # ... with 843 more rows
+
+Now let’s plot the network of these relationships between words, using
+the [`igraph`](http://igraph.org/r/) package by Gábor Csárdi and Támas
+Nepusz and [`ggraph`](https://github.com/thomasp85/ggraph) package by
+Thomas Lin Pedersen.
+
+``` r
+library("igraph")
+library("ggraph")
+
+set.seed(2016)
+
+word_cors %>%
+  dplyr::filter(correlation > .35) %>%
+  graph_from_data_frame() %>%
+  ggraph(layout = "fr") +
+  geom_edge_link(aes(edge_alpha = correlation), show.legend = FALSE) +
+  geom_node_point(color = "lightblue", size = 5) +
+  geom_node_text(aes(label = name), repel = TRUE) +
+  theme_void()
+```
+
+![](/img/blog-images/2018-05-10-onboarding-social-weather/unnamed-chunk-14-1.png)
+
+This figure gives a good sample of things discussed in reviews. Despite
+our efforts filtering words specific to issues, some of them remain very
+specific, such as country/city/location that are very frequent in
+`ropenaq` review.
+
+#### How positive is onboarding?
+
+Using sentiment analysis, we can look at how positive comments are.
+
+``` r
+sentiments %>%
+  dplyr::group_by(role) %>%
+  skimr::skim(sentiment)
+```
+
+    ## Skim summary statistics
+    ##  n obs: 11553 
+    ##  n variables: 6 
+    ##  group variables: role 
+    ## 
+    ## Variable type: numeric 
+    ##               role  variable missing complete    n  mean   sd   min p25
+    ##             author sentiment       0     4823 4823 0.07  0.21 -1.2    0
+    ##  community_manager sentiment       0       97   97 0.13  0.21 -0.41   0
+    ##             editor sentiment       0     1521 1521 0.13  0.22 -1.63   0
+    ##              other sentiment       0      344  344 0.073 0.2  -0.6    0
+    ##           reviewer sentiment       0     4768 4768 0.073 0.21 -1      0
+    ##  median  p75  max     hist
+    ##   0     0.17 1.84 <U+2581><U+2581><U+2582><U+2587><U+2581><U+2581><U+2581><U+2581>
+    ##   0.071 0.23 1    <U+2581><U+2581><U+2587><U+2585><U+2582><U+2581><U+2581><U+2581>
+    ##   0.075 0.25 1.13 <U+2581><U+2581><U+2581><U+2581><U+2587><U+2586><U+2581><U+2581>
+    ##   0     0.2  0.81 <U+2581><U+2581><U+2582><U+2587><U+2582><U+2582><U+2581><U+2581>
+    ##   0     0.17 1.73 <U+2581><U+2581><U+2587><U+2585><U+2581><U+2581><U+2581><U+2581>
+
+``` r
+summary(sentiments$sentiment)
+```
+
+    ##     Min.  1st Qu.   Median     Mean  3rd Qu.     Max. 
+    ## -1.63200  0.00000  0.00000  0.07961  0.18353  1.84223
+
+``` r
+sentiments %>%
+  dplyr::filter(!role %in% c("other", "community_manager")) %>%
+  ggplot(aes(role, sentiment)) +
+  geom_boxplot(fill = "salmon") +
+  hrbrthemes::theme_ipsum(base_size = 16,
+                          axis_title_size = 16,
+                          strip_text_size = 16)
+```
+
+![Sentiment of onboarding review threads by
+line](/img/blog-images/2018-05-10-onboarding-social-weather/unnamed-chunk-15-1.png)
+
+Generally lines are quite positive (positive mean, zero 25th-quantile),
```

Comment:
> lines are quite positive (positive mean, zero 25th-quantile),

I'm naive but maybe other readers are too. Could you briefly note what the salmon coloured bars indicate? The plot still doesn't look "quite positive" to me

Reply:
> are generally not negative 

Looks like I made you go to the opposite extreme ;-) Maybe you could say "are generally positive"